.github/workflows/ci.yml

```
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         version:
-          - '1.3'
+          - '1.7'
```

Did you mean
```suggestion
          - '1.6'
```
? This is running the tests twice on v1.7